daprdocs/content/en/reference/resource-specs/resiliency-schema.md

```diff
@@ -0,0 +1,60 @@
+---
+type: docs
+title: "Resiliency spec"
+linkTitle: "Resiliency spec"
+weight: 3000
+description: "The basic spec for a Dapr resiliency resource"
+---
+
+The `Resiliency` Dapr resource allows you to define and apply fault tolerance resiliency polcies. Resiliency specs are applied with the Dapr sidecar starts. 
```

Comment:
```suggestion
The `Resiliency` Dapr resource allows you to define and apply fault tolerance resiliency policies. Resiliency specs are applied with the Dapr sidecar starts. 
```
This sentence is unclear "Resiliency specs are applied with the Dapr sidecar starts. "